osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -336,5 +409,40 @@ private void load()
                 });
             }
         }
+
+        private partial class NewCommentEditor : CommentEditor
+        {
+            [Resolved]
+            public Bindable<CommentableType> CommentableType { get; set; }
+
+            [Resolved(name: cid)]
+            public BindableLong CommentableId { get; set; }
```

Comment:
Does this need to use bindables? As far as I can see these can just be passed down normally via ctor?

Reply:
Editor isn't recreated when ID changes and container is reloaded, so this needs to be tracked. Passing bindables via ctor won't work because this data will be needed in reply boxes too.
It could be simpler if i resolve `CommentsContainer` itself and take values from it at moment of submit, will try it.